index.html

```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
+  <header class="about-me"> 
+    <h2> About myself</h2>
+  </header>
+   
+
+   <p class="abt-p">
+    Hello I’m a software developer! I can help you build a product , feature or website. 
+    Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+   </p>
+
+   <h2 class="lets-connect">let's connect</h2>
+
+   <div class="link">
+    <ul class="media-icons">
+      <li>
+        <a href="#">
+          <img src="./images/Twitter.png" alt="twitter">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Linkedin icon.png" alt="linkedin">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Mail.png" alt="mail">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Github.png" alt="github">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Vector.png" alt="vector">
+
+  
+
+  </div>
+    <h1 class="resume">Get my resume</h1>
```

Comment:
- [ ] The `Get my resume` is going to be used by your website visitors to get access to your resume, so I'll suggest you use a `button` or `a` here. :+1: 
- [ ] Also I noticed your design does not match the given Figma design, I'll suggest you go through the design one more time and make sure your design looks like the Figma design as that's part of the requirement for this milestone. :+1: 
- [ ] I'll suggest you implement the button states here `hover, active`. :+1:  


- YOUR DESIGN:
![Screenshot from 2023-05-04 19-24-23](https://user-images.githubusercontent.com/107632780/236296674-f580c8c7-9a56-4c1e-800f-fccd91c5e08d.png)

- FIGMA DESIGN:
![Screenshot from 2023-05-04 19-24-06](https://user-images.githubusercontent.com/107632780/236296757-091926b2-d0f8-40d4-98a0-2d805336ce4c.png)